src/System.Management.Automation/engine/SessionStateFunctionAPIs.cs

```diff
@@ -195,7 +195,7 @@ private bool IsFunctionVisibleInDebugger(FunctionInfo fnInfo, CommandOrigin orig
             var runspace = this.ExecutionContext.CurrentRunspace;
             if ((runspace != null) &&
                 (runspace.InNestedPrompt || (runspace.Debugger?.InBreakpoint == true)) &&
-                (fnInfo.DefiningLanguageMode.HasValue && (fnInfo.DefiningLanguageMode != this.ExecutionContext.LanguageMode)))
+                (fnInfo.DefiningLanguageMode.GetValueOrDefault() != this.ExecutionContext.LanguageMode))
```

Comment:
```suggestion
                (fnInfo.DefiningLanguageMode.GetValueOrDefault() != this.ExecutionContext.LanguageMode && fnInfo.DefiningLanguageMode.HasValue))
```

Reply:
This change would be required to preserve the preserve the existing logic. Making a change here might improve codegen, at the expense of unnatural C#.